.github/workflows/mdtf_tests.yaml

```diff
@@ -0,0 +1,162 @@
+          - os: macos-latest
+            json-file: "tests/github_actions_test_ubuntu_set1.jsonc"
+          - os: macos-latest
+            json-file-set2: "tests/github_actions_test_ubuntu_set2.jsonc"
+          - conda-root: /usr/share/miniconda3
+            json-file: "tests/github_actions_test_macos_set1.jsonc"
+          - conda-root: /Users/runner/minconda3
+            json-file: "tests/github_actions_test_ubuntu_set1.jsonc"
+          - conda-root: /usr/share/miniconda3
+            json-file-set2: "tests/github_actions_test_macos_set2.jsonc"
+          - conda-root: /Users/runner/minconda3
+            json-file-set2: "tests/github_actions_test_ubuntu_set2.jsonc"
+      max-parallel: 2
+    steps:
+    - uses: actions/checkout@v2
+    - name: Download Miniconda 3
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        miniconda-version: "latest"
+        python-version: 3.8
+    - name: Verify miniconda
+      run: |
+        conda info -a
+    - name: Install XQuartz if macOS
+      if: ${{ matrix.os == 'macos-latest' && matrix.conda-root == '/Users/runner/miniconda3' }}
+      run: |
+        echo "Installing XQuartz"
+        brew install --cask xquartz
+    - name: Set environment variables
+      run: |
+        echo "POD_OUTPUT=$(echo $PWD/../wkdir/GFDL.Synthetic)" >> $GITHUB_ENV
+    - name: Install Conda Environments
+      run: |
+        # install mamba (https://github.com/mamba-org/mamba) for faster dependency resolution
+        echo "Installing Mamba"
+        conda install mamba -n base -c conda-forge
+        echo "Installing Conda Environments"
+        # MDTF-specific setup: install all conda envs
+        ./src/conda/conda_env_setup.sh --all --mamba --conda_root ${{matrix.conda-root}}
+    - name: Generate Model Data
+      run: |
+        cd ../
+        # clone the mdtf_test_data repo
+        git clone https://github.com/jkrasting/mdtf_test_data.git
+        # Defining dependencies in Conda env is faster than pip
+        conda activate _env_synthetic_data
+        # install the test data software
+        cd mdtf_test_data
+        pip install .
+        # generate the data
+        ./scripts/gfdl_synthetic.py
+        ./scripts/ncar_synthetic.py
+        cd ../
+        mkdir wkdir
+        echo "Directories and files in the current directory:"
+        ls
+    - name: Get Observational Data for Set 1
+      run: |
+        echo "${PWD}"
+        cd ../
+        echo "Available Space"
+        df -h
+        # attempt FTP data fetch
+        # allow 20 min for transfer before timeout; Github actions allows 6 hours for individual
+        # jobs, but we don't want to max out resources that are shared by the NOAA-GFDL repos.
+        #curl --verbose --ipv4 --connect-timeout 8 --max-time 1200 --retry 128 --ftp-ssl --disable-epsv --ftp-pasv -u "anonymous:anonymous" ftp://ftp.gfdl.noaa.gov/pub/jessica.liptak/example_obs_data.tar --output example_obs_data.tar
```

Comment:
@wrongkindofdoctor - we should request some permanent ftp space that won't be subject to the sweeper.

Reply:
Agreed. I can submit a helpdesk ticket for permanent space under the role account.

Reply:
Good catch -- I wasn't aware that files in users' public FTP directories get swept.

Reply:
The yaml has been updated to point to the obs_data files in the permanent FTP directory